docs/installation/cloud/overview.md

```diff
@@ -26,11 +26,11 @@ To install on a cloud provider:
 
 2. Decide which OS you want to run on the cloud host.
```

Comment:
How about `# Choose how to install` ? It works fine on menu --- the understand is kind a implied as you'd have to understand some stuff to choose.